sdk/servicebus/Azure.Messaging.ServiceBus/src/Amqp/AmqpConnectionScope.cs

```diff
@@ -954,19 +974,33 @@ public override void Dispose()
             Uri endpoint,
             string[] audience,
             string[] requiredClaims,
-            TimeSpan timeout)
+            TimeSpan timeout,
+            string identifier)
         {
+            string uri = endpoint.AbsoluteUri;
+            ServiceBusEventSource.Log.RequestAuthorizationStart(identifier, uri);
```

Comment:
This seems redundant to the existing `AmqpLinkAuthorizationRefresh` series  (line 865).    Am I overlooking something?

Reply:
The difference is that the refresh event is only logged on refreshes, whereas the new logs are written on every request (including the initial one). The new event is also verbose as opposed to Informational.